Sources/Brave/Frontend/Sync/BraveCore/History/HistoryAPIExtensions.swift

```diff
@@ -53,10 +54,20 @@ extension BraveHistoryAPI {
     }
   }
   
-  func deleteAll(completion: @escaping () -> Void) {
+  func deleteAll(includeScreenTimeHistory: Bool = true, completion: @escaping () -> Void) {
```

Comment:
I think we can remove the parameter. The only place used deleteAll is in history screen and I dont think we will ever need to call deleteAll without needing to delete screen time history.